### DISCUSSION TITLE
Getting stuck in client-secure console

### DISCUSSION THREAD

**Discussion by kismanhong** [cockroachdb/cockroach#64463] [Q&A] [answered]
I try deploy cockroachdb in Rancher(secure mode), when i try to execute shel it always stuck (see screenshot). I have 3 nodes.
<img width="1329" alt="Screen Shot 2021-04-30 at 16 33 20" src="https://user-images.githubusercontent.com/472404/116677210-242bab00-a9d2-11eb-8c3d-38477f088af0.png">

**Comment by jordanlewis** [comment, marked as answer]
There's not enough information here to help you, sorry! It looks like there is a networking issue between your client and server.
